release-notes/2022-12/README.md

```diff
@@ -47,6 +47,8 @@ The 2022-12 release bundle for SystemLink Enterprise has been published to <http
 - `notification 0.1.0`
     - Newly included chart
     - You must configure a secret for MongoDB credentials in `notification.secrets.mongodb`
+- `sl-jupyterhub 0.2.0`
+    - The default configuration doesn't allow anymore user pods to establish outbound connections to private IPs (for example accessing SystemLink APIs that are exposed on a private IP from Jupyter Notebooks or the Jupyter terminal). To allow this, you must explicitly set `jupyterhub.singleuser.networkPolicy.egressAllowRules.privateIPs=true`.
```

Comment:
```suggestion
    - JupyterHub user pods may no longer establish outbound connections to private IP addresses by default. To enable this, you must explicitly set `jupyterhub.singleuser.networkPolicy.egressAllowRules.privateIPs=true`.
```

Reply:
I applied @MelissaHilliard 's suggestion. Let me know if you think the text should still be modified